src/algorithm/verifySolidAccessToken.ts

```diff
@@ -87,13 +82,13 @@ export async function verifySolidAccessToken(
   const accessToken = {
     header: protectedHeader,
     payload,
-    signature: accessTokenParts[2],
+    signature: solidJwt.jwsSignature,
```

Comment:
It might be a bit weird that this `accessToken` has a mix of decoded and encoded values.

Reply:
You're quite right and I don't know that I actually need to really construct this object.

I'll refactor the whole validation to make things clearer as part of another PR, it would be a big change.